### ISSUE TITLE
5.5.2-alpha1 bug in PhoneNumberCapabilities json deserialization

### ISSUE THREAD

**Issue by gbronzer** [twilio/twilio-csharp#380]
**Version: 5.5.2-alpha1**
**API Subdomain (api/REST/messaging):**

Using Newtonsoft.Json v10.0.3

### Code Snippet
```c#
var number = PhoneNumberResource.Create(serviceSid, numberSid);
```

### Exception / Log
```
Twilio.Exceptions.ApiException: Error converting value \"MMS\" to type 'Twilio.Types.PhoneNumberCapabilities'. Path 'capabilities[0]', line 1, position 95. ---> Newtonsoft.Json.JsonSerializationException: Error converting value \"MMS\" to type 'Twilio.Types.PhoneNumberCapabilities'. Path 'capabilities[0]', line 1, position 95. ---> System.ArgumentException: Could not cast or convert from System.String to Twilio.Types.PhoneNumberCapabilities.\r\n   at Newtonsoft.Json.Utilities.ConvertUtils.EnsureTypeAssignable(Object value, Type initialType, Type targetType)\r\n   at Newtonsoft.Json.Utilities.ConvertUtils.ConvertOrCast(Object initialValue, CultureInfo culture, Type targetType)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EnsureType(JsonReader reader, Object value, CultureInfo culture, JsonContract contract, Type targetType)\r\n   --- End of inner exception stack trace ---\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EnsureType(JsonReader reader, Object value, Cultu
reInfo culture, JsonContract contract, Type targetType)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(IList list, JsonReader reader, JsonArrayContract contract, JsonProperty containerProperty, String id)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetP
ropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)\r\n  
 at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)\r\n   at Twilio.Rest.Messaging.V1.Service.PhoneNumberResource.FromJson(String json)\r\n   --- End of inner exception stack trace ---\r\n   at Twilio.Rest.Messaging.V1.Service.PhoneNumberResource.FromJson(String json)\r\n   at Twilio.Rest.Messaging.V1.Service.PhoneNumberResource.Create(CreatePhoneNumberOptions options, ITwilioRestClient client)\r\n   at Twilio.Rest.Messaging.V1.Service.PhoneNumberResource.Create(String pathServiceSid, String phoneNumberSid, ITwilioRestClient client)\r\n   at Jackrabbit.Services.SMS.Providers.TwilioProvider.InitializeAccount() in C:\\jrdev\\trunk\\JRNet\\app\\Jackrabbit.Services\\SMS\\Providers\\TwilioProvider.cs:line 250"

```

### Steps to Reproduce
1. Create a messaging service
2. Attempt to add a number with Voice, SMS, MMS support to the service

**Comment by efossier** [comment]
@gbronzer Thanks for writing in, definitely looks like a bug on our end in the deserialization logic. I've created a ticket to address it.

**Comment by efossier** [comment]
@gbronzer This has been fixed in `5.5.3-alpha2` https://github.com/twilio/twilio-csharp/releases
